app/bundles/CampaignBundle/Controller/CampaignController.php

```diff
@@ -143,6 +149,140 @@ public function cloneAction(Request $request, $objectId)
         return $this->cloneStandard($request, $objectId);
     }
 
+    /**
+     * Export an entity.
+     *
+     * @return JsonResponse|BinaryFileResponse|Response
+     */
+    public function exportAction(ExportHelper $exportHelper, CampaignModel $campaignModel, int $objectId)
```

Comment:
Please avoid comments that are obvious from the code. Use real return types over comments whenever possible.
```suggestion
    public function exportAction(ExportHelper $exportHelper, CampaignModel $campaignModel, int $objectId): JsonResponse|BinaryFileResponse|Response
```

Reply:
fixed